extra/utils/validation.js

```diff
@@ -0,0 +1,57 @@
+let users = [
+     {
+          traineeEmail : "shikhar.singh@successive.tech",
+          reviewerEmail : "naresh.kumar@tcs.tech",
+     },
+     
+     {
+          traineeEmail : "niyati.pavar@tcs.tech",
+          reviewerEmail : "nilesh.agarwal@successive.tech",
+     },
+
+     {
+          traineeEmail : "sam.sharma@wipro.tech",
+          reviewerEmail : "vimal.avasti@wipro.tech",
+     },
+
+     {
+          traineeEmail : "shyam.bansal@successive.tech",
+          reviewerEmail : "@successive.techamit.kumar",
+     },
+
+     {
+          traineeEmail : "rohan.kumawat@google.tech",
+          reviewerEmail : "suraj.kumar@google.tech",
+     },
+
+]
+
+function validateEmail(user_email)
+{
+     return /^[a-zA-Z0-9+_.-]+@successive.tech+$/.test(user_email);     
+}
+
+function validateUsers(users)
```

Comment:
Hi shikhar, as per the AC, your function need to return the result either true or false, here you are just logging the response. please correct it